### ISSUE TITLE
Option toggle to auto close tabs on bookmark

### ISSUE THREAD

**Issue by clemera** [teddy-gustiaux/default-bookmark-folder#232]
# This is a Feature Request

**Is your feature request related to a problem? Please describe.**
When I clean up my tab bar I often would like to stash tabs away to bookmarks.

**Describe the solution you would like**
A toggle in the toolbar icon to enable auto close on bookmark. I would enable the option and
start stashing tabs away. When done I would disable the toggle again.

**Comment by tomtom** [comment]
Yes, please. This would make default-bookmark-folder a even better replacement for the onetab addon.
